tests/screenshot/specs/AgateCopperDay-specs.js

```diff
@@ -0,0 +1,8 @@
+const {runTest} = require('@enact/ui-test-utils/utils');
+const Page = require('./AgatePage');
+
+runTest({
+	testName: 'Agate Copper Day',
+	Page: Page,
+	skin: 'copper-day'
```

Comment:
I think we'll need to update the spec runner here to pass variants down.  It may also need some additional changes in `ui-test-utils` to pass the variants down appropriately.  It's possible we can do this from the apps here, but I'll have to dig in first.